js/apps/admin-ui/src/components/dynamic/UrlComponent.tsx

```diff
@@ -0,0 +1,42 @@
+import { FormGroup } from "@patternfly/react-core";
+import { useTranslation } from "react-i18next";
+
+import { HelpItem } from "ui-shared";
+import { Controller, useFormContext } from "react-hook-form";
+import type { ComponentProps } from "./components";
+import { FormattedLink } from "../external-link/FormattedLink";
+import { convertToName } from "./DynamicComponents";
+
+export const UrlComponent = ({
+  name,
+  label,
+  helpText,
+  defaultValue
+}: ComponentProps) => {
+  const { t } = useTranslation();
+  const { control } = useFormContext();
+  
+  return (
+    
+    <FormGroup
+      label={t(label!)}
+      fieldId={name!}
+      labelIcon={<HelpItem helpText={t(helpText!)} fieldLabelId={`${label}`} />}
+      isRequired={false}
+      className="keycloak__identity-providers__saml_link"
```

Comment:
I confirm it's needed otherwise the link is not aligned correctly - I took your suggestion and created a new class just for this (in the beginning I tried to reuse existing CSS classes but I see your point in keeping things clean and separated)